crates/bevy_ecs/src/event.rs

```diff
@@ -283,8 +283,8 @@ impl<'a, 'w, 's, E: Event> IntoIterator for &'a mut EventReader<'w, 's, E> {
 ///     // which allows one to do all kinds of clever things with type erasure, such as sending
 ///     // custom events to unknown 3rd party plugins (modding API).
 ///     //
-///     // NOTE: the event won't actually be sent until commands get flushed
-///     // at the end of the current stage.
+///     // NOTE: the event won't actually be sent until commands get applied during
+///     // apply_system_buffers.
```

Comment:
Can you make this link to `apply_system_buffers`?

Reply:
I don't think I can in this case since it's in a doc-test code block, not the main body of a docstring

Reply:
Oh, right.